### PR TITLE
Remove SynonymousTo

### PR DESCRIPTION
**Pull request by Mihaiii**
For https://github.com/StarcounterApps/Simplified/issues/95
Related to https://github.com/StarcounterApps/Simplified/pull/96

### CODE REVIEW

**Comment by Mihaiii** [comment]
@miyconst 
I can test it myself, I just need a confirmation that it wasn't left aside on purpose.

File: Authorization/Database/PermissionSomebodyGroup.cs [hunk 1]
```diff
@@ -10,10 +10,8 @@ namespace Starcounter.Authorization.Database
     [Database]
     public class PermissionSomebodyGroup : Relation
     {
-        [SynonymousTo(nameof(WhatIs))]
```
Comment (miyconst) [inline Authorization/Database/PermissionSomebodyGroup.cs]: By simply removing the attribute you are creating a new independent property, which is not correct. Please see my property implementations in Simplified.

**Comment by Mihaiii** [comment]
I made a commit with the proper implementation.

File: Authorization/Database/PermissionSomebodyGroup.cs [hunk 2]
```diff
@@ -10,8 +10,33 @@ namespace Starcounter.Authorization.Database
...
+            get
+            {
+                return this.WhatIs as PermissionToken;
+            }
+            set
+            {
+                this.WhatIs = value;
+            }
+        }
+        /// <summary>
```
Comment (miyconst) [inline Authorization/Database/PermissionSomebodyGroup.cs]: Please add a space line between the methods declaration.

File: README.md [hunk 3]
```diff
@@ -167,8 +167,28 @@ public class InvoiceRow : Something
...
+        get
+        {
+            return this.From as Invoice;
+        }
+        set
+        {
+            this.From = value;
+        }
+    }
+    public InvoiceRow InvoiceRow
```
Comment (miyconst) [inline README.md]: Please add a space line between the methods declaration and the inline comments.

File: Authorization/Database/PermissionSomebodyGroup.cs (hunk 2 above)
Comment (Mihaiii) [reply Authorization/Database/PermissionSomebodyGroup.cs]: Done.

File: README.md (hunk 3 above)
Comment (Mihaiii) [reply README.md]: Done.

Comment (miyconst) [reply README.md]: Please add a space line between the methods declaration **and the inline comments**.

**Pull request merged**